pkg/openfeaturetest/testprovider_test.go

```diff
@@ -84,79 +82,77 @@ func TestParallelSingletonUsage(t *testing.T) {
 func TestTestAwareProvider(t *testing.T) {
 	taw := NewTestAwareProvider()
 
-	memProvider := memprovider.NewInMemoryProvider(
-		map[string]memprovider.InMemoryFlag{
-			"ff-bool": {
-				State:          memprovider.Enabled,
-				DefaultVariant: "variant_1",
-				Variants: map[string]any{
-					"variant_1": true,
-				},
+	flags := map[string]memprovider.InMemoryFlag{
```

Comment:
Users are dealing with flags now, not providers, which is conceptually lighter, IMO.